ui/app/components/secret-edit.js

```diff
@@ -235,6 +238,27 @@ export default Component.extend(FocusOnInsertMixin, {
       set(this.modelForData, 'secretData', this.secretData.toJSON());
     },
 
+    handleWrappedClick() {
+      if (this.isV2) {
+        const { path } = this.model.versions.objectAt(this.model.currentVersion - 1);
```

Comment:
I think `this.modelForData` will be the secret for v1 and the secret version for v2 probably better to use that here since we could also be on a page where it's an older version and getting the last one wouldn't work then.